packages/lib/src/common/utils/index.js

```diff
@@ -652,6 +652,17 @@ class Utils {
       styles.cubeImages &&
       styles.cubeRatio === '100%/100%';
   }
+
+  getAriaAttributes(props) {
+    const { proGalleryRole, proGalleryRegionLabel } = props;
```

Comment:
This could be destructed in the arguments that the function gets. so it will expect an object with these keys and we can get rid of "props".  
getAriaAttributes({ proGalleryRole, proGalleryRegionLabel }) {}

Reply:
👍